getTrapForcesForSingleParticle.m

```diff
@@ -1,4 +1,9 @@
 function total_force = getTrapForcesForSingleParticle(particlePosition, trapPositions, A, s)
 %Calculate force from all optical traps on a particle
-f = (A./s.^2) .* (particlePosition-trapPositions) .* exp((-0.5*(particlePosition-trapPositions)./s).^2);
-total_force = sum(f,1);
\ No newline at end of file
+%f = (A./s.^2) .* (particlePosition-trapPositions) .* exp(-((0.5*(particlePosition-trapPositions)./s).^2));
+%total_force = sum(f,1);
+%%
```

Comment:
The convenstion for variable names is without underscores _. So the x force for instance should be denoted fx, as in the other functions